src/simplify_algebra.cpp

```diff
@@ -682,29 +662,21 @@ struct find_add_convs
             {
                 if(a_op.stride < b_op.stride)
                 {
-                    auto n = compute_stride_factor(b_op, a_op);
+                    int64_t n = static_cast<int64_t>(compute_stride_factor(b_op, a_op));
```

Comment:
Why is `int64_t` needed?

Reply:
I had thought n has to be int64_t  type to initialize the value. Looks like it does not have to be. changed it.